### PR TITLE
8.0 Add UNECE modules

### PR DESCRIPTION
**Pull request by alexis-via**
These modules are required for electronic invoicing (CII, ZUGFeRD, UBL, etc...)

### CODE REVIEW

**Comment by alexis-via** [comment]
Runbot doesn't take into account the file  oca_dependencies.txt ; that's why runbot is red. But travis takes it into account already, that's why it is green.

**Comment by pedrobaeza** [comment]
Dependency added on runbot. Are you sure this project is the best suited for this module? I think _account-invoicing_ is a better host.

**Comment by alexis-via** [comment]
@pedrobaeza I don't know which project is the most suited. I thought it was this one, because there are a LOT of unece codifications and what I propose is only a small part of it, the part which i need for CII and UBL invoices... but we may need other UNECE codifications in the future of other needs.

File: account_payment_unece/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```
Comment (pedrobaeza) [inline account_payment_unece/__openerp__.py]: Use short headers like https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py

File: account_payment_unece/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+
+{
+    'name': 'Account Payment UNECE',
+    'version': '8.0.0.1.0',
```
Comment (pedrobaeza) [inline account_payment_unece/__openerp__.py]: 8.0.1.0.0

**Comment by pedrobaeza** [comment]
@alexis-via, you're right that the project is very transversal, so it can fit here. As I was thinking about your account_invoice_import modules, I said the other, but this is bigger and different than that.

**Comment by gurneyalex** [comment]
:+1:

**Comment by bguillot** [comment]
:+1:

**Comment by alexis-via** [comment]
Can someone with the super-powers merge this please ?

**Comment by alexis-via** [comment]
@pedrobaeza I know you have the commit rights on all OCA projects, could you merge this please ?

**Comment by pedrobaeza** [comment]
@alexis-via, sorry for not attending on time, but @sebastienbeau has made it instead of me. Thanks!

**Pull request merged**